src/content/docs/about/getting-started.mdx

```diff
@@ -0,0 +1,273 @@
+    subtitle="Learn how to set a Target to define how Daytona manages your development environment."
+    href="/configuration/providers#managing-targets"
+    expanded
+  />
+</DocumentList>
+
+## Choosing the Default IDE
+
+Daytona allows you to connect to your development environment using a variety of [IDEs](/usage/ide), including popular options like [Visual Studio Code (VSCode)](/usage/ide#vs-code), JetBrains IDEs, and [Terminal SSH](/usage/ide#terminal-ssh). By setting the default IDE, you can automatically open your development environment in your preferred environment, streamlining your workflow and reducing setup time.
+
+<br />
+
+<CardGrid gridNumber={9} mobileGridNumber={2} colsMinWidth='64px' colsMinWidthMobile='98px'>
+ <ImageCard icon={VSlogo} link="/usage/ide#vs-code" imageAlt="Visual Studio logo" />
+ <ImageCard  imageDark={PClogo} link="/usage/ide#jetbrains-pycharm-professional" imageAlt="PyCharm logo" />
+ <ImageCard  imageDark={IJlogo} link="/usage/ide#jetbrains-intellij-idea-ultimate" imageAlt="IntelliJ logo" />
+    <ImageCard  imageDark={WSlogo} link="/usage/ide#jetbrains-webstorm" imageAlt="WebStorm logo" />
+    <ImageCard  imageDark={GOlogo} link="/usage/ide#jetbrains-goland" imageAlt="GoLand logo" />
+    <ImageCard  imageDark={CLlogo} link="/usage/ide#jetbrains-clion" imageAlt="CLion logo" />
+    <ImageCard  imageDark={RDlogo} link="/usage/ide#jetbrains-rider" imageAlt="Rider logo" />
+    <ImageCard  imageDark={RMlogo} link="/usage/ide#jetbrains-rubymine" imageAlt="RubyMine logo"/>
+</CardGrid>
+
+For more details on setting the default IDE and related commands, use the `daytona ide` command to explore options and manage your configurations.
+
+<DocumentList>
+<DocumentListItem
+    title="Set Default IDE"
+    subtitle="Learn how to set your default IDE to automatically open your development environment."
+    href="/configuration/git-providers"
+    expanded
+  />
+</DocumentList>
+
+## Creating your Workspace
+
+Once you have completed the initial setup and configuration steps, you can [create your Workspace](/usage/workspaces#create-a-workspace) using Daytona. A Workspace represents an isolated development environment where you can manage your code, build projects, and collaborate with team members.
+
+Creating a Workspace involves defining the necessary configurations, such as the Git repository URL and selecting the appropriate [Git Provider](/configuration/git-providers). Daytona simplifies this process by providing a user-friendly interface to set up and manage your Workspaces efficiently.
+
+To create a Workspace, use the `daytona create` command and follow the prompts to configure your Workspace settings. Once created, you can access your Workspace and interact with your codebase.
+
+<DocumentList>
+<DocumentListItem
+    title="Create a Workspace"
+    subtitle="Learn how to create your Workspace using Daytona."
+    href="/usage/workspaces"
+    expanded
+  />
+</DocumentList>
+
+## Selecting a Builder
```

Comment:
This is an advanced option and I would label it as such. I would also note that we detect the builder type automatically if left untouched.